src/components/formRow/formRow.comp.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import { orderBy } from 'natural-orderby';
 import { toast } from 'react-toastify';
+import { Multiselect } from 'multiselect-react-dropdown';
```

Comment:
Why not using a more popular (and maintained) library such as `react-select`?
https://react-select.com/home

Reply:
I had problems with others importing the `finalOptions` object and found this one to contain additional parameters that were useful like the `selectionLimit` for multi.  Also, it allowed me to use the same component for single and multi-select.